docs/en/solutions/How_to_Deploy_nmstate.md

```
@@ -0,0 +1,261 @@
+checkmodule -M -m -o nmstate-networkmanager-dbus.mod nmstate-networkmanager-dbus.te
+semodule_package -o nmstate-networkmanager-dbus.pp -m nmstate-networkmanager-dbus.mod
+sudo semodule -i nmstate-networkmanager-dbus.pp
+```
+
+### Installation Steps
+
+#### Install Using Offline Installation Script
+
+**Prerequisites:** Ensure the installation script is available in the attachments directory.
+
+**Run Installation Script**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)